tests/e2e/testfiles/powerflex_alt_vals_4_values.csv

```diff
@@ -1,4 +1,4 @@
 parameter name, grep option for paramater value, parameter value,resource to describe,number of occurences of parameter name
-Image,,dellemc/csi-vxflexos,csm,1
+Image,,quay.io/dell/container-storage-modules/csi-vxflexos,csm,1
```

Comment:
Are we sticking with quay or using dockerhub? I see you reverting the change to quay in the operatorconfig but using quay here.

Reply:
quay.io, I updated the operatorconfig change you mentioned